### DISCUSSION TITLE
Ultima - Angular Standalone

### DISCUSSION THREAD

**Discussion by tomy29** [primefaces/community#2688] [PrimeNG Templates]
Hi, 
is there a way to make Ultima template works with an Angular application that uses standalone components? 
Is the migration to standalone planned?
thank you.

**Comment by Mathieu-Lichtsteiner** [comment]
Any NgModule can be referenced in the imports array of a standalone component. If you need specific advice, please explain your problems in more detail.

Angular standalone components are planned, as you can see in the following discussion:
https://github.com/orgs/primefaces/discussions/923#discussion-6075903
